IMPLEMENTATIONS.md

```diff
@@ -1,10 +1,9 @@
 ### Implementations
 
-Below is a list of known tooling implementing the 3.0.0 specification. Because
-the 3.0.0 specification has not yet been released, please consult the details of
-any projects listed below for notes about stability and roadmap.  The process 
+Below is a list of known tooling that implements the 3.0.0 specification. Because
```

Comment:
@krishahn Since this PR changes lines 3-5, would you please consider doing one sentence per line?